wp-content/themes/pub/wporg-learn-2024/patterns/front-page-content.php

```diff
@@ -7,6 +7,16 @@
 
 ?>
 
+<!-- wp:group {"layout":{"type":"flex","flexWrap":"wrap"}} -->
+<div class="wp-block-group">
+	<!-- wp:search {"showLabel":false,"placeholder":"<?php esc_html_e( 'Search for resources', 'wporg-learn' ); ?>","width":100,"widthUnit":"%","buttonText":"<?php esc_html_e( 'Search', 'wporg-learn' ); ?>","buttonPosition":"button-inside","buttonUseIcon":true} /-->
```

Comment:
Is "Search for resources" the language we are expecting?

Reply:
I like "Search for resources", what do you think @kathrynwp?